<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1502</width>
    <height>841</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(160, 225, 150, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QListWidget" name="listWidget_Pcf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>341</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>BGLR Evening</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>YUANTA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>YUANTA_2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FUBON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CATHAY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGLR_Samsung</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>----------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>US_Harvest_China</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>----------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK Morning</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_Samsung</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_ChinaAMC_Bond</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_GlobalX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_Nikko</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_Nikko63</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>----------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_Noon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_PingAn</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_ChinaAMC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_WisdomTree</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_EASY_COMO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_EASY_FI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_EASY_JBEM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>----------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_Samsung-ICE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HK_Nikko63_intra</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BetaShares</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Test</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidget_pathPcf">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>711</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_producePcf">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>2. Produce PCF</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_Clear">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear window</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ArchiveMails">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <italic>false</italic>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Archive Mails</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Date">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Root Path (not taken into acct if Folder start  \\ ):</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DevEnv">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>OneDrive</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_folderRoot">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_comparison">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>761</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashDotLine</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>4</number>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>70</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>15</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="push_sendPCF">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>3. Send Mail PCF</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_loadPcf">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>Retrieve PCF path
 (if App closed)</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_DisplayMail">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>314</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Display Mail</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_downloadFiles">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>1. Download Files</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_DisplayDoc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Display Doc</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_OpenParamCSV">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open Param.csv</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_OpenParamMailCSV">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open Param Mail</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_ComparePCF">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Compare w. Manual</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Error">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>500</y>
     <width>501</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;Error Return:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Result">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>331</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;Main Figures:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_KillExcel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>Kill Excel 
 (for Ghost session)</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_Minimize">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>10</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimize</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_DB">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>10.233.6.147</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D1PRDSOLADB.infocloud.local</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidget_filePresentFTP">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>580</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="push_sendFTP">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>510</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>3. Send FTP</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MailType">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>225</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1. Send Client Mail</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. Confirm With Provider</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3. Send Late Alert Email</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
